lib/plugins/exchange/cctx/cctx.js

```diff
@@ -0,0 +1,57 @@
+var ccxt = require('ccxt')
+const coinUtils = require('../../../coin-utils')
+const _ = require('lodash/fp')
+const common = require('../../common/cctx')
+
+function trade (side, account, cryptoAtoms, fiatCode, cryptoCode, exchangeName) {
+  try {
+    const exchange = setUpExchange(account, exchangeName)
+    const symbol = common.verifyCurrencies(exchangeName, fiatCode, cryptoCode)
+    const amount = coinUtils.toUnit(cryptoAtoms, cryptoCode).toFixed(8)
+    if (exchangeName === 'itbit') {
+      return exchange.fetchOrderBook(symbol)
+        .then(orderBook => {
+          return exchange.createOrder(symbol, 'limit', side, amount, calculatePrice(side, amount, orderBook), { walletId: account.walletId })
+        })
+    }

```

Comment:
I think we can remove the { } wrapped around this return statement. From what I gather, our styling convention dictates that if we have only a return statement inside an if, we don't use curly braces

Reply:
Hey, this is only for ifs that can be put in a single line. If it goes multiline {} is required (even by standard)